sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -46,6 +46,27 @@
       config:
         gradle_target: 'sdks:java:extensions:sql:expansion-service:shadowJar'
 
+- type: renaming
+  transforms:
+    'ReadFromKafka': 'ReadFromKafka'
+  config:
+    mappings:
+      'ReadFromKafka':
+        'schema': 'schema'
+        'consumer_config': 'consumerConfigUpdates'
+        'format': 'format'
+        'topic': 'topic'
+        'bootstrap_servers': 'bootstrapServers'
+        'confluent_schema_registry_url': 'confluentSchemaRegistryUrl'
+        'confluent_schema_registry_subject': 'confluentSchemaRegistrySubject'
```

Comment:
do we need the naming of `confluent_schema_registry_...` ?  what if using a different schema registry, like redpanda's?

Reply:
maybe confluent_... is tied to something specific elsewhere?